### PR TITLE
Use the minute rate limit in API access JWTs

### PR DESCRIPTION
**Pull request by dempe**
@ThorArakis Sorry, the indentation got changed...

### CODE REVIEW

File: kong/plugins/sm-account-rate-limiting/handler.lua [hunk 1]
```diff
@@ -52,112 +52,113 @@ end
...
-  if usage then
-    -- Adding headers
-    for k, v in pairs(usage) do
-      ngx.header["X-Account-Rate-Limit-Month"] = v.limit
-      ngx.header["X-Account-Rate-Limit-Remaining-Month"] = math.max(0, (stop == nil or stop == k) and v.remaining - 1 or v.remaining) -- -increment_value for this current request
+    local account_id = parse_claim(jwt, "account_id")
+    local rate_limits = parse_claim(jwt, "ratelimit")
+
+    if is_empty(rate_limits) then
+        return responses.send_HTTP_BAD_REQUEST("Could not find ratelimit.month in JWT.  This plugin works only with monthly, account limits.")
```
Comment (ThorArakis) [inline kong/plugins/sm-account-rate-limiting/handler.lua]: Fix the error message.

**Comment by ThorArakis** [comment]
np, if you append ?w=1 to the end of a github PR it strips most of the whitespace. It's my favorite :)

**Comment by ThorArakis** [comment]
seems simple enough to me. Are there tests that need to get fixed?

**Comment by dempe** [comment]
Grr... Yes, there are broken tests now. :(

**Comment by ThorArakis** [comment]
👍

**Pull request merged**